### ISSUE TITLE
Ground connected to body?

### ISSUE THREAD

**Issue by KoviRobi** [Spirit532/studio_mic#1]
Hi,

Thanks for the schematics and instructions, I've managed to build my own :) I wouldn't be able to come up with the design/know how to tune it but I did manage to understand how it works, with the help of the internet. I was just wondering, is it intentional that the PCB ground traces are not connected to the body? I did do the sanding of the body, it was all electrically connected from one end to the other, but I had huge amounts of hum (about twice the signal strength) until I connected the ground of the XLR to the body. Maybe the mounting holes could be plated and connected to the ground if this is necessary, or should this not have been necessary and maybe I did something wrong?

Thanks,
Rob

**Comment by Spirit532** [comment]
Note the GNDREF near the XLR connector, J1, and capacitors C91 and C92. GNDREF is already connected through the XLR wiring(on pin 1), which should ground the body _at the connector_, mechenically. C91 and C92 must also be mounted on the connector pins directly. There's a chance you misnumbered the connector while soldering, or the body isn't electrically connected with the XLR connector, due to some manufacturing difference with what was done on mine.

**Comment by KoviRobi** [comment]
Yeah I think it's that the body isn't connected to the XLR connector -- there is actually a separate tab on the XLR connector that connects to the screw/body (and this isn't pin 1, nor is it connected to it) so that would be it, thanks!
